lib/galaxy/container/docker_swarm_manager.py

```diff
@@ -0,0 +1,560 @@
+"""
+Docker Swarm mode management
+"""
+from __future__ import absolute_import
+
+import argparse
+import errno
+import logging
+import os
+import subprocess
+import sys
+import time
+
+try:
+    import daemon
+    import daemon.pidfile
+    import lockfile
+except ImportError:
+    daemon = None
+
+try:
+    import galaxy   # noqa: F401 this is a test import
+except ImportError:
+    sys.path.insert(0, os.path.abspath(os.path.join(
+        os.path.dirname(__file__),
+        os.pardir,
+        os.pardir)))
+
+from galaxy.config import (
+    configure_logging,
+    find_path,
+    find_root,
+)
```

Comment:
I think I'd prefer moving `docker_swarm_manager.py` rather than moving the entire `container` package. And also, we should probably standardize whether packages in galaxy should tend to include a plural form or not.